Library/SharedFunctions.swift

```diff
@@ -683,11 +683,62 @@ public func estimatedShippingText(for reward: Reward, selectedShippingRule: Ship
   let formattedMax = String(format: "%.0f", estimatedMax)
 
   // TODO: Update string with translations [mbl-1667](https://kickstarter.atlassian.net/browse/MBL-1667)
-  let shippingString: String = formattedMin == formattedMax
-    ? "About $\(formattedMin)"
-    : "About $\(formattedMin)-$\(formattedMax)"
+  let estimatedShippingString: String = formattedMin == formattedMax
+    ? "$\(formattedMin)"
+    : "$\(formattedMin)-$\(formattedMax)"
 
-  return shippingString
+  return estimatedShippingString
+}
+
+public func estimatedShippingConversionText(
+  _ project: Project,
+  _ reward: Reward,
+  selectedShippingRule: ShippingRule
+) -> String {
+  guard project.stats.needsConversion else { return "" }
+
+  /// Make sure the current reward has shipping rules and that one of them matches the selected shipping rule (from the locations dropdown).
+  guard let shippingRules = reward.shippingRules,
+        let selectedShippingRule = shippingRules
+        .first(where: { $0.location.country == selectedShippingRule.location.country })
+  else {
+    return ""
+  }
+
+  guard let estimatedMin = selectedShippingRule.estimatedMin?.amount.rounded(.towardZero),
+        let estimatedMax = selectedShippingRule.estimatedMax?.amount.rounded(.towardZero),
```

Comment:
Why are you rounding these? They're getting rounded by the format function anyways. Rounding first and then converting just makes the final result less accurate.